lib/sdn_epc/forwarder.ex

```diff
@@ -0,0 +1,55 @@
+defmodule SdnEpc.Forwarder do
+  use GenServer
+
+  # Client API
+
+  def start_link do
+    res = GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
+  end
+
+  def subscribe_messages_from_switch(datapatch_id, type) do
+    GenServer.cast __MODULE__,
+    {:handle_switch_msg, datapatch_id: datapatch_id, type: type}
+  end
+
+  def open_ofp_channel(switch_id, ip, port, version) do
+    GenServer.call __MODULE__,
+    {:open_of_channel, %{switch_id: switch_id, ip: ip,
+                         port: port,version: version}}
```

Comment:
No space after coma. You should add something like ebert (https://ebertapp.io/) to automatically check you code.